tests/collections/common/loss_inputs.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Copyright 2018-2020 William Falcon
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections import namedtuple
+
+import torch
+
+from .pl_utils import BATCH_SIZE, NUM_BATCHES, NUM_CLASSES
```

Comment:
No relative imports. Please do an absolute import everywhere.

Reply:
More so than this, this import is cyclic. Please copy the constants from that file here onto this file to avoid it.

Reply:
I have a problem with using absolute imports. Command `pytest` fails with import error.
```
ImportError while importing test module '/home/apeganov/NeMo/tests/collections/common/test_metrics.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../anaconda3/envs/translation_new_py37/lib/python3.7/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/collections/common/test_metrics.py:22: in <module>
    from tests.collections.commmon.loss_inputs import _all_num_measurements_are_zero, _no_zero_num_measurements, _some_num_measurements_are_zero
E   ModuleNotFoundError: No module named 'tests.collections.commmon'
```
@titu1994 could you please help?

This import is not cyclic in my opinion.

Reply:
Ok seems like pytest wants relative imports. As to cyclic import, since tests are running and only constants are being imported, I guess it's fine. 

Thanks for looking into this c